.changeset/eight-beds-smash.md

```diff
@@ -0,0 +1,65 @@
+---
+'@leafygreen-ui/date-picker': major
+---
+
+Changes the behavior of segments.
+
+## New Behavior
```

Comment:
@TheSonOfThomp, if you think there is a better way to organize this, please let me know!

Reply:
I like this! Maybe move the old/new callouts to each section (i.e. Backspace: old / new  etc.) to make it easier to compare?